src/tests/multihost/alltests/test_misc.py

```diff
@@ -353,3 +353,48 @@ def test_0006_getent_group(self, multihost,
         assert "group-2@example1:*:20002:user-2@example1," \
                "user-4@example1,user-6@example1," \
                "user-8@example1" in cmd.stdout_text
+
+    @pytest.mark.tier1
+    def test_0007_getent_admproxy(self, multihost, backupsssdconf):
+        """
+        :title: 'getent passwd adm@proxy' doesn't
+         return anything when 'cache_first = True' option is used with
+         nss i.e. lookup with the fully-qualified name of a user or
+         group will fail if the requested object is not already in the
+         cache.
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2013294
+        :id: 9ff64ee0-255d-46ac-bf0a-b022eaad463e
+        :customerscenario: false
+        :steps:
+            1. Configure SSSD with nss having cache_first = True.
+            2. restart SSSD with empty cache.
+            3. Check 'getent passwd adm@proxy' output.
+        :expectedresults:
+            1. Should succeed
+            2. Should succeed
+            3. 'getent passwd adm@proxy' should return identity lookup
+            of user adm.
```

Comment:
indent 'of' to start under single quote before getent

Reply:
I've made the change and committed. Thanks.